vision/roadmap.md

```diff
@@ -101,7 +99,29 @@ Apps will be able to leverage an example server to subscribe users for push noti
 
 ## General Availability ✅
 
-Ready for production use. Check out [docs](https://xmtp.org/docs/client-sdk/javascript/concepts/intro-to-sdk), [tutorials](https://xmtp.org/docs/client-sdk/javascript/tutorials/build-an-xmtp-hello-world-app), and the [quickstart example](https://xmtp.org/docs/client-sdk/javascript/tutorials/quickstart).
+### Support for JavaScript and React apps
```

Comment:
Given roadmap page is for a concise overview of all things at XMTP, could we:

- Reduce link outs to improve readability?
- Reduce repetition by removing/changing 'support for' verbiage